erpnext/setup/doctype/item_group/item_group.py

```diff
@@ -150,7 +150,8 @@ def adjust_qty_for_expired_items(data):
 		if item.get('has_batch_no') and item.get('website_warehouse'):
 			stock_qty_dict = get_qty_in_stock(
 				item.get('name'), 'website_warehouse', item.get('website_warehouse'))
-			qty = stock_qty_dict.stock_qty[0][0] if stock_qty_dict.stock_qty else 0
+			stock_qty = stock_qty_dict.stock_qty
```

Comment:
I'm not sure if this would work, since it seems like you're only storing the info in a variable, but the logic remains the same.

Can you please add / update the test case for this?

Reply:
@netchampfaris I also doubt it will fix the error. Can you confirm?

Reply:
@netchampfaris, I think you're looking at an older bench. The fix you're trying to make was already made [here](https://github.com/frappe/erpnext/commit/0c8411d085b9f0f6e3db645d24ae90503631d313#diff-f0a387cdb305471e74e523ecc4e646ac). I think this PR can be safely closed.